packages/components/src/components/contract-card/contract-card-items/contract-card-header.jsx

```diff
@@ -28,6 +28,24 @@ const ContractCardHeader = ({
     const { underlying, multiplier, contract_type, shortcode, purchase_time, date_expiry, tick_count, is_sold } =
         contract_info;
 
+    const contract_type_list_info = [
+        {
+            value: 'Multiplier',
+            checkContractType: isMultiplierContract,
+            displayed_trade_text: `x${multiplier}`,
+        },
+        {
+            value: 'Turbos',
+            checkContractType: isTurbosContract,
+            displayed_trade_text: getSubType(contract_type),
+        },
+    ];
+
+    const displayed_trade_param =
+        contract_type_list_info.find(contract_type_item_info =>
+            contract_type_item_info.checkContractType(contract_type)
+        )?.displayed_trade_text || '';
+
     return (
         <>
```

Comment:
help to update this to `React.Fragment` in your PR.

Reply:
Sure, thank you